diagnostics/app/views/feedback.scala.html

```diff
@@ -0,0 +1,36 @@
+@(page: model.Page, emailParams: String, uri: Option[String])(implicit request: RequestHeader)
+
+@main(page){ }{
+<div class="l-side-margins">
```

Comment:
good idea, thanks for the pointer @gidsg , actually I just based it on the "content is expired" page for now.  If the feature starts getting used, I'll appreciate some more help in following your suggestion.  If it doesn't get used I don't want to spend a long time on the first version - hence the lightweight version of everything.